frontend/vue.config.js

```diff
@@ -23,5 +23,13 @@ module.exports = {
       .end()
       .use('vue-svg-loader')
       .loader('vue-svg-loader')
+  },
+  pluginOptions: {
+    i18n: {
+      locale: 'de',
+      fallbackLocale: 'en',
+      localeDir: 'locales',
+      enableInSFC: true
+    }
```

Comment:
Du verwendest diese Einstellungen gar nirgends, könnte man das entfernen? Oder in i18n.js auf diese Optionen zurückgreifen? https://cli.vuejs.org/config/#pluginoptions

Reply:
Die Optionen werden ziemlich sicher von `vue-cli-plugin-i18n` verarbeitet, denke nicht, dass man die einfach entfernen kann. Optionen wiederverwenden finde ich aber gut, so dass wir 'de' als Standardsprache nun an einem Ort definieren müssen.